components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -214,6 +214,14 @@ be lost if SCREAM_HACK_XML is not enabled.
     <!-- CLD fraction -->
     <cldFraction inherit="atm_proc_base"/>
 
+    <!-- nudging -->
+    <nudging inherit="atm_proc_base">
+      <nudging_filename type="array(string)">NONE</nudging_filename>
```

Comment:
We no longer have to put a placeholder for arrays. For an empty default, you can simply do
```
<nudging_filename type="array(string)"/>
```
which avoids maintenance burden of handling the default invalid string...